src/components/DashBoard/TableOrders/TableAllOrders.jsx

```diff
@@ -178,7 +211,8 @@ const OrdersTableAll = () => {
                                             </td>
                                         )}
                                         <td data-label="Cotizado">
-                                            {user.type === "admin" || user.type === "superadmin" ? (
+                                            {user.type === "admin" ||
+                                            user.type === "superadmin" ? (
                                                 order?.cotizated === null ? (
```

Comment:
i think this is a bit repetitive, maybe you can create a constant with this ternary like this:
const userIsAdmin = user.type === "admin" || user.type === "superadmin";
So the result is:
{userIsAdmin ? ( ...